src/components/Analyze/AnalyzeContext.tsx

```diff
@@ -0,0 +1,37 @@
+import { useAnalyzeStore } from '@/components/store/AnalyzeStore'
+import { createContext } from 'react'
+
+interface analyzeDataType {
+	category: string
+	detail: string
+	price: number
+	date: string
+}
+
+interface AnalyzeContextType {
+	consumptionData?: analyzeDataType[]
+	totalPrice?: number
+}
```

Comment:
이 부분에서의 interface 정의와 `AnalyzeStore`의 interface 정의가 중복됩니다.
types 폴더에 아래와 같은 형식으로 타입을 정의해서 가져다 쓰는게 좋을 것 같습니다.
```suggestion
export interface analyzeDataType {
	category: string
	detail: string
	price: number
	date: string
}

export interface AnalyzeContextType {
	consumptionData?: analyzeDataType[]
	totalPrice?: number
}
```